scripts/api_dump_generator.py

```diff
@@ -698,7 +698,11 @@
 std::ostream& dump_text_{funcName}(ApiDumpInstance& dump_inst, {funcReturn} result, {funcTypedParams})
 {{
     const ApiDumpSettings& settings(dump_inst.settings());
-    settings.stream() << "Thread " << dump_inst.threadID() << ", Frame " << dump_inst.frameCount() << ":\\n";
+    settings.stream() << "Thread " << dump_inst.threadID() << ", Frame " << dump_inst.frameCount();
+    if(settings.showTimestamp())
+        settings.stream() << ", Time " << dump_inst.current_time_since_start().count() << " μs\\n";
+    else 
+        settings.stream() << "\\n";
```

Comment:
The formatting in the new code is subtly different than the original: the original `Thread` line had a trailing colon, while the new `Thread` line doesn't have a trailing colon in either case.

I'm not sure whether we promise the output format of this line anywhere.  But I'm concerned that changing it could break people relying on the original format.

Reply:
Thats an oversight on my part.
Adding the colon back is simple enough, and if they are looking for a colon, then there is no good reason to not have it. Putting timestamps in is still a breaking change (if the feature is enabled) so parsers still need to update. Or disable timestamps, in which their code should still work fine.

Reply:
Fixed and made timestamps disabled by default.